### ISSUE TITLE
AttributeError: module 'twitter' has no attribute 'Api'

### ISSUE THREAD

**Issue by datascienceconsultingllc** [bear/python-twitter#611]
AttributeError: module 'twitter' has no attribute 'Api' 

This is similar to other issues and I've read through them all, but I am still having issues. I understand there is a twitter and a python-twitter library. I've tried pip uninstall twitter pip install python-twitter and I've also tried pip install python-twitter==3.1.  **Update** I've also tried to install directly from git through anaconda prompt and still running into issues. 

Basically it works on my first install of Anaconda.  When I close it down and reopen it for a second time I am getting this error. I have uninstalled Anaconda and then reinstalled it and my code worked until I closed and reopened it.  

`#import packages and connect to Twitter API
import twitter
import pandas as pd

api = twitter.Api(consumer_key='hidden',
consumer_secret='hidden',
access_token_key='hidden',
access_token_secret='hidden')`

**Comment by datascienceconsultingllc** [comment]
I think I found my error. I believe when I'm using this with the TwitterBot package there is something going on with the Twitter vs. Python-Twitter packages in the background. I believe the TwitterBot package calls and installs the Twitter package.

**Comment by PokestarFan** [comment]
Yes I am pretty sure that for whatever reason the other twitter package overwrites the twitter module. It depends on what was installed most recently. Usually, you should be able to call `pip uninstall twitter` to get rid of it.

Also @datascienceconsultingllc you should close this issue if you know what the error is.

**Comment by meloman-a** [comment]
I had the same problem, because of incorrect intalled module "twitter" instead of "python-twitter".
My steps: just delete "twitter" module and install "python-twitter".
